video_xblock/tests/mocks/base.py

```diff
@@ -0,0 +1,156 @@
+"""
+Base Video Xblock mocks.
+"""
+import json
+from copy import deepcopy
+from collections import OrderedDict
+from mock import Mock
+import requests
+
+
+from video_xblock.exceptions import VideoXBlockMockException
+
+
+class ResponseStub(object):
+    """
+    Dummy ResponseStub class.
+    """
+
+    def __init__(self, **kwargs):
+        """
+        Delegate kwargs to class properties.
+        """
+        for key, val in kwargs.items():
+            setattr(self, key, val)
+
+    @property
+    def text(self):
+        """
+        Make response compatible with requests.Response.
+        """
+        return getattr(self, 'body', '')
+
+    @property
+    def content(self):
+        """
+        Make response compatible with requests.Response.
+        """
+        return getattr(self, 'body', '')
+
+    def get(self, key):
+        """
+        Allow to fetch data from response body by key.
+        """
+        body = getattr(self, 'body', '')
+        if body:
+            try:
+                return json.loads(body)[key]
+            except KeyError:
+                pass
+
+
+class BaseMock(Mock):
+    """
+    Base custom mock class.
```

Comment:
@polesye yes, it's possible to use it but this lib will rather extend base mock but not replace it